docs/main/plugins.mdx

```diff
@@ -0,0 +1,23 @@
+# Plugins SDK
+
+Plugins allow you to host interactive, collaborative, and efficient meetings.
+You can use plugins like Remote Browser, YouTube, Miro, Excalidraw, and many more to collaborate when having the Dyte meeting.
+
+Plugins SDK allows developers to build, test and deploy Plugins.
+Currently, this SDK is available through Beta Testing.
+Please fill out this form if you would like to try Plugins SDK and build Plugins for Dyte.
```

Comment:
Please fill out this form if you would like to try out our Plugin SDK and build plugins for Dyte.

Reply:
Updated